src/store/worker/workerSlicte.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+import { api } from "../../api_client/api";
+import { IWorkerAvailabilityResponse, WorkerAvailabilityResponse } from "./worker.zod";
+
+const initialState: IWorkerAvailabilityResponse = {
+  status: true,
+  job_detail: undefined,
+  queue_can_accept_job: false,
+};
+
+const workerSlicte = createSlice({
```

Comment:
Typo here and in file name.

Reply:
whoops. Updated it.

Also reverted all library updates since apparently there's a lot of TS issues which aren't quickly resolvable

  * https://github.com/i18next/next-i18next/issues/1795
  * https://github.com/cookpete/react-player/issues/1406

Should all be good now.